src/import/sst_service.rs

```diff
@@ -1240,6 +1307,27 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
         RawChunk,
         new_raw_writer
     );
+
+    fn suspend_import_rpc(
```

Comment:
prefer name it `suspend_import`

Reply:
Changing a RPC name needs to change all the repos we involved. We have just rename all `deny` to `suspend`. For now I think we'd better delay all renaming stuff to all behavior has been defined and we are about to merge all 3 prs.